Kitodo-API/src/main/java/org/kitodo/api/validation/longtermpreservation/LtpValidationCondition.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.validation.longtermpreservation;
+
+import java.util.List;
+
+/**
+ * An immutable, no-setters beans implementing the validation condition interface.
+ */
+public class LtpValidationCondition implements LtpValidationConditionInterface {
+
+    private String property;
+    private LtpValidationConditionOperation operation;
+    private List<String> values;
+    private LtpValidationConditionSeverity severity;
+
+    /**
+     * Create a new (immutable, no-setters) validation condition.
```

Comment:
The first sentence of a Javadoc should be a verbal phrase (VP), that is a sentence where the subject is omitted, so that it begins with the verb in the third-person singular form (here, "creates"). @solth, you can still merge the PR, but it would be nice to clean it up in a cleanup PR.